docs/source/file_download.rst

```diff
@@ -0,0 +1,76 @@
+=================================
+File Download on the Data Portal
```

Comment:
One thing with this: look closely at the document once this is rendered. Sometimes our RST renderer eats spaces. I think the way you've done this here is probably fine, since you're keeping to a single line continuous text/paragraphs. But it's always good to double check it, since it doesn't appear the same way on local as it does on live/prod envs.